README.md

```diff
@@ -19,16 +19,16 @@ Chrome has been shipping its own nonstandard segmentation API called `Intl.v8Bre
 let segmenter = new Intl.Segmenter("fr", {granularity: "word"});
 
 // Get an iterator over a string
-let iterator = segmenter.segment("Ceci n'est pas une pipe");
+let iterator = segmenter.break("Ceci n'est pas une pipe");
 
 // Iterate over it!
-for (let {segment, breakType} of iterator) {
-  console.log(`segment: ${segment} breakType: ${breakType}`);
+for (let {index, breakType} of iterator) {
```

Comment:
I thought the goal was for one sweeping reset. Do you really want a separate PR to remove `breakType`?

Reply:
Why would we remove breakType? I don't understand the motivation for a sweeping reset.

Reply:
> **I don't understand the motivation for a sweeping reset.**

Because many aspects of the current interface were based upon a muddled model that at least partially presumed iteration over segments, such as `breakType`, the interpretation of `following`/`preceding` arguments, and the semantics of `%SegmentIteratorPrototype%.index`.

> **Why would we remove breakType?**

https://github.com/tc39/proposal-intl-segmenter/issues/44#issuecomment-484907925
> The current text does a poor job of defining what `breakType` is. Possible values seem to describe segments rather than boundaries, and it is not specified to _which_ boundary-adjacent segment they correspond with. This is especially confusing for backwards iteration—what is the proper value of `breakType` after `(new Intl.Segmenter("fr", {granularity: "word"})).segment("Ceci n'est pas une pipe").preceding(8)`? There's also the issue of a missing definition for "_numbers, letters, kana characters, ideographic characters, etc_" and "_sentence terminator ('.', '?', '!', etc.)_".
> 
> I am in favor of removing `breakType` because it is easy for consumers to check the break-preceding code unit at `index - 1` on their own, but if `breakType` or a renamed equivalent remains then it needs a better and more complete specification.